onboarding.md

```diff
@@ -205,12 +205,14 @@ needs to be pointed out separately during the onboarding.
   automatically closed.
 * Label your pull request with the `doc`, `notable-change`, and `fast-track`
   labels.
-* Run CI on the PR. Use the `node-test-pull-request` CI task.
+* Run CI on the pull request. Use the `node-test-pull-request` CI task.
 * After two Collaborator approvals for the change and two Collaborator approvals
-  for fast-tracking, land the PR.
-* Leave a comment in the PR: `Please 👍 this comment to approve fast-tracking`.
+  for fast-tracking, land the pull request.
+* Leave a comment in the pull request:
+  `Please 👍 this comment to approve fast-tracking`.
```

Comment:
Not exactly related to this PR, but shouldn't update this now that GH Actions adds the comment automatically?

Reply:
Oh, you beat me to it 😅 https://github.com/nodejs/node/pull/39410

Reply:
Yeah, I figured there might be some back-and-forth about whether to remove or alter that, and I didn't want that to stall the rest of the changes in this pull request, so I put it in its own pull request.